monitorcontroller/monitorcontroller.go

```diff
@@ -70,9 +69,17 @@ func New(
 	config *config.Config, monitoringSender MonitoringSender,
 ) (monitor *MonitorController, err error) {
 	monitor = &MonitorController{
-		monitoringSender:    monitoringSender,
-		monitoringQueue:     make([]cloudprotocol.NodeMonitoringData, 0, config.Monitoring.MaxOfflineMessages),
-		monitoringQueueSize: config.Monitoring.MaxOfflineMessages,
+		monitoringSender:   monitoringSender,
+		offlineMessages:    make([]cloudprotocol.Monitoring, 0, config.Monitoring.MaxOfflineMessages),
+		sendMessageEvent:   make(chan struct{}, 1),
+		maxMessageSize:     config.Monitoring.MaxMessageSize,
+		currentMessageSize: 0,
```

Comment:
No need to initialize fields with default values

Reply:
fixed